components/recents/WindowHeight.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState, useEffect } from "react";
+
+function getWindowDimensions() {
+  const height = innerHeight 
```

Comment:
what is 'innerHeight'?
Also why not just remove this function altogether and just use the innerHeight variable?